tests/test_build.py

```diff
@@ -273,27 +273,25 @@ def test_logo_template_rejected(sphinx_build_factory) -> None:
         sphinx_build_factory("base", confoverrides=confoverrides).build()
 
 
-@pytest.mark.parametrize(
-    "align,klass",
-    [
-        ("content", ("col-lg-3", "col-lg-9", "me-auto")),
-        ("left", ("", "", "me-auto")),
-        ("right", ("", "", "ms-auto")),
-    ],
-)
-def test_navbar_align(align, klass, sphinx_build_factory) -> None:
+def test_navbar_align_default(sphinx_build_factory) -> None:
```

Comment:
any reason you dropped the parameters to test the navbar align ? I have the feeling that "content" set manually is not tested anymore and neither "left"

Reply:
that was probably a rebasing mistake, I didn't to it on purpose. will fix.